src/core/client/stream/index.html

```diff
@@ -5,20 +5,193 @@
     <meta charset="utf-8" />
     <meta http-equiv="Content-type" content="text/html; charset=utf-8" />
     <meta name="viewport" content="width=device-width" />
+    <style>
+      /* styles for skeleton representation of components before React loads */
+      .load-flex {
+        display: flex;
+      }
+      .load-flex-spaceBetween {
+        justify-content: space-between;
+      }
+      .load-flex-wrap {
+        flex-wrap: wrap;
+      }
+      .load-flex-alignItems-end {
+        align-items: flex-end;
+      }
+      .load-flex-column {
+        flex-direction: column;
+      }
+      .load-height-xs {
+        height: 1rem;
+      }
+      .load-height-s {
+        height: 2rem;
+      }
+      .load-height-m {
+        height: 3rem;
+      }
+      .load-height-l {
+        height: 8rem;
+      }
+      .load-width-xs {
+        width: 4rem;
+      }
+      .load-width-s {
+        width: 8rem;
+      }
+      .load-width-m {
+        width: 12rem;
+      }
+      .load-width-l {
+        width: 18rem;
+      }
+      .load-width-full {
+        width: 100%;
+      }
+      .load-bgGrey100 {
+        background-color: #f4f7f7;
+      }
+      .load-bgGrey200 {
+        background-color: #eaeff0;
+      }
+      .load-bgGrey300 {
+        background-color: #cbd1d2;
+      }
+      .load-rMargin-s {
+        margin-right: 0.5rem;
+      }
+      .load-tMargin-s {
+        margin-top: 0.5rem;
+      }
+      .load-tMargin-m {
+        margin-top: 1rem;
+      }
+      .load-lMargin-s {
+        margin-left: 0.5rem;
+      }
+      .load-lMargin-auto {
+        margin-left: auto;
+      }
+      .load-hr {
+        border: 1px solid #eaeff0;
+      }
+      @media (max-width: 639px) {
```

Comment:
I've seen a few of these show up lately. I've done them myself.

I've seen them in `ConversationModalQuery.css` and `UserHistoryDrawer.css`.

Would you mind adding this one to `sharedVariables.ts` as something like `mediaStops` and call this whichever size you think makes sense (maybe `lg` or similar)?

That way we can start consolidating these into named variables to start cleaning up the `magic number` factor a bit. 😄